catanatron_rust/src/state_vector.rs

```diff
@@ -102,6 +102,11 @@ pub fn player_devhand_slice(num_players: u8, color: u8) -> std::ops::Range<usize
     start..start + 5
 }
 
+pub fn player_played_devhand_slice(num_players: u8, color: u8) -> std::ops::Range<usize> {
+    let start = PLAYER_STATE_START_INDEX + num_players as usize + 11 + (color as usize * 15);
+    start..start + 4
```

Comment:
Should these magic numbers be constants? What does the 11, 15, etc... mean? Thanks!